### PR TITLE
real data import

### PR DESCRIPTION
**Pull request by manuel-hegner**

### CODE REVIEW

File: import.sql [hunk 1]
```diff
@@ -0,0 +1,87 @@
+CREATE TEMP TABLE data_import (
```
Comment (vseufe) [inline import.sql:1]: Not sure if project root really is a good place for this file :/

Comment (vseufe) [inline import.sql:1]: The script also runs if the database is not empty and then causes exceptions in the backend

**Review by vseufe — CHANGES_REQUESTED** [review]

File: import.sql (hunk 1 above)
Comment (manuel-hegner) [reply import.sql:1]: I am not sure what you mean. Is data_import not empty? It is a TEMP table, but it could make sense to drop it afterwards.

Comment (manuel-hegner) [reply import.sql]: moved to `useful scripts/`

[the remark this answers is not in the document]
Comment (manuel-hegner) [reply import.sql]: it is used as a script to manually import data

File: src/main/resources/db/data/V1.9.0__Insert_real_sample_data.sql [hunk 2]
```diff
@@ -0,0 +1,135 @@
+INSERT INTO public.course VALUES
+	(1, 'tarent, Bonn', 'EXTERNAL', false, '2020-09-07 00:00:00', '2020-09-08 00:00:00', 'www.feedback-people.de', 'REMOTE', 'Tamara', 'Vertriebler; Führungskräfte;Trainer', 'Rhetorik', 'Feedback People', 'Präsentationstechniken und Storytelling sowie Auftreten und Gesprächsführung erlernen.', 'kostenpflichtig', 'SEMINAR'),
```
[the remark this answers is not in the document]
Comment (manuel-hegner) [reply src/main/resources/db/data/V1.9.0__Insert_real_sample_data.sql]: new version just uses the import script, so no IDs are used

**Review by vseufe — APPROVED** [review]

**Pull request merged**